apps/heft/src/cli/HeftActionRunner.ts

```diff
@@ -76,6 +76,35 @@ export function initializeHeft(
   terminal.writeVerboseLine('');
 }
 
+let _cliAbortSignal: AbortSignal | undefined;
+export function ensureCliAbortSignal(terminal: ITerminal): AbortSignal {
```

Comment:
Might consider migrating this over to operation-runner.

Reply:
@dmichon-msft - you want to do that in a follow-up?

Reply:
Sure.